stylesheet/main.css

```diff
@@ -107,41 +124,31 @@ h2 {
     color: transparent;
 }
 
-div paperclip{position: absolute;
+.paperclip{position: absolute;
              width :50px;
              height:50px;
-            top:10px;
-            right-top: 7px;
+            top: -10px;
+            right: -7px;
+            background-image: url('../images/paperclip.png');
             background-size :contain;
-            padding-top:1em;
-            padding-bottom:0.5em;
-            margin-right:1em;
-            margin-left:1em;
-
-    
 }
 
-My Three Favourites NBA stars{
-    
+.favourites {
+    clear: both;
 }
+
 header{
-    float:right;
-    opacity: 0.9;
-    border-top: 5px solid;
+    border-top: 4px solid rgba(0,0,0,0.1);
     margin-right:-2em;
     margin-left:-2em;
-    
-    
-}
-.box{
-     float:left;
 }
+
 nav{
     margin-right:2em;
     margin-left:2em;
      margin-top:-4px;
-    padding:2em;
-    border-bottom:1px;opacity: 0.7;
+    border-bottom: 1px dotted rgba(0,0,0,0.3);
+    overflow: auto;

```

Comment:
This declaration is what causes the nav element to take on the height of its floated contents